.travis.yml

```diff
@@ -1,15 +1,13 @@
 language: go
 go:
 - 1.x
-dist: trusty
-services:
-- docker-ce
+dist: xenial
+go_import_path: github.com/google/keytransparency
```

Comment:
Love the `go_import_path` but since this a separate step, please put this in a separate commit.  (It can still be in this same PR).